src/components/cms/BrowseSkill/BrowseSkill.js

```diff
@@ -809,41 +816,76 @@ class BrowseSkill extends React.Component {
 
             {!this.props.loadingSkills ? (
               <div style={styles.container}>
-                <div>{renderCardScrollList}</div>
                 {metricsHidden ? (
                   <div>
-                    <div id={'page-filter'}>
-                      {renderSkillCount}
-                      {skills.length > 10 && (
-                        <div id={'pagination'}>
-                          <FormControl
-                            style={{ width: '150px', marginTop: '15px' }}
-                          >
-                            <InputLabel>Skills per page</InputLabel>
-                            <Select
-                              value={this.props.entriesPerPage}
-                              onChange={this.handleEntriesPerPageChange}
+                    <Grid
+                      container
+                      spacing={3}
+                      direction={isMobile ? 'column-reverse' : 'row'}
+                    >
+                      <Grid
+                        item
+                        alignItems="center"
+                        sm={6}
+                        style={{
+                          textAlign: 'center',
+                          padding: isMobile ? '10px' : '25px',
+                          fontSize: isMobile ? '14px' : '16px',
+                        }}
+                      >
+                        {renderSkillCount}
+                      </Grid>
+                      <Grid
+                        item
+                        sm={6}
+                        alignContent="flex-end"
+                        style={{ alignItems: isMobile ? 'center' : 'left' }}
+                      >
+                        {skills.length > 10 && (
+                          <div>
+                            <FormControl
+                              style={{
+                                width: isMobileView(350) ? '120px' : '145px',
```

Comment:
Why does isMobileView take in a param. It should do so.

Reply:
@akshatnitd This was done because iphone 5 has 320px width while it works perfectly fine for all phones above 350px width.